projects/angular-showcase/src/app/business/business-examples/textexpand-showcase/textexpand-showcase.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sbb-textexpand-showcase',
+  templateUrl: './textexpand-showcase.component.html',
+  styleUrls: ['./textexpand-showcase.component.scss']
```

Comment:
You can remove styleUrls and the corresponding css file, when the css file is empty anyway.